src/dbg/runtime.ts

```diff
@@ -1269,6 +1269,38 @@ or define the location manually with the launch.json->mapFile setting`
         return this._getScope(this._currentPosition);
     }
 
+    private static async which(path: string): Promise<string> {
+        // On Windows, the PATHEXT variable contains an ordered list of file extensions
+        // for executable (e.g., .exe, .com, .bat) that we need to additionally probe.
+        const pathExt = process.env.PATHEXT?.split(";") ?? [];
```

Comment:
Is there some other way to do this? Windows should already be doing this by default. I would feel more comfortable if this logic was coming from a library.